azure/template.json

```
@@ -88,6 +88,9 @@
         "keyVaultCertificateName": {
             "type": "string",
             "defaultValue": ""
+        },
+        "ipSecurityRestrictions": {
+           "type": "array"
```

You'll want this to be optional, it won't be used in dev/testing environments
```suggestion
           "type": "array",
           "defaultValue": []
```